### PR TITLE
First portfolio work

### PR DESCRIPTION
**Pull request by sambeck87**
#### A HTML file was created
- Images and links were added
- Content was added

#### A CSS file was created
- Font family, font color, and font size were customized
- Background color was changed
- All elements were positioned

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  <div class="general">
+    <header>
+      <div class="mobile">
+        <div id="time">9:41</div>
+        <div id="icons">
+          <img src="./Images/Cellular Connection.png" alt="signal">
+          <img src="./Images/Wifi.png" alt="wifi">
+         <img src="./Images/Battery.png" alt="batery">
+       </div>
+     </div>
```
Comment (Qoosim) [inline index.html]: - [ ] Kindly remove this part of the code. It is not part of your mobile design. It is just device peculiarities.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+     <div class="menu">
+        <p>Jonh Doe</p>
```
Comment (Qoosim) [inline index.html]: - [ ] This is like a logo to your website. And every logo has a link to the home page of the site. Kindly change the `<p>` tag to `<a>` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        <a href="#"><img src="./Images/Union.png" alt="menu"></a>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required to wrap your hamburger inside the `<nav>` tag for semantic reason. Kindly wrap your hamburger inside a `<nav>` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+     <h4>I'am a Software Developer</h4>
+      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded<br>don't hestiate to contact me.</p>
+
+      <nav class="media">
+       <a href="https://github.com/sambeck87"><img src="./Images/social/Disabled.png" alt="github"></a>
+       <a href="https://www.linkedin.com/in/sandro-israel-hern%C3%A1ndez-zamora-899386a4/"><img src="./Images/social/Disabled-1.png" alt="linkedin"></a>
+        <a href="#"><img src="./Images/social/Enabled.png" alt=""></a>
+        <a href="https://twitter.com/sambeck4488"><img src="./Images/social/Disabled-2.png" alt=""></a>
+        <a href="#"><img src="./Images/social/Disabled-3.png" alt=""></a>  
+        </nav>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required that the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Kindly apply this suggestion to this part of your project.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @sambeck87, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Nice UI. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] It is required to produce a close replica of the Figma template given. This includes font-size, font-type, background-color, color, margin, padding among others. There are some glaring differences in your design compare with the guiding template. Kindly look into below screenshots of both your design and Figma design and work your design Figma design.

### Your Design for Font-type

- [ ] Kindly make use of the font-type use on the Figma template on your design for both your header and headline 

your-design


### Figma Design For Font-type

temp

### Your Design For Spacing

- [ ] In order to produce a close replica of the guiding template, you must consider almost every properties available on the guiding template on your design. The space between your title and your introduction is too wide compare to what we have on the guiding template. Kindly look into this and fix it.

your-spacing


### Figma Design For Spacing

temp-spacing

**Comment by sambeck87** [comment]
Hi @Qoosim.
I tried to change the font-family, but I don't know how can I get the same type. I think I need a link to add it to my HTML file to import that specific font-family. Could you help me by telling me where I can get the link, please?
I appreciate all your help.

**Comment by Qoosim** [comment]
Hi @sambeck87,
Oh sorry for the late response. You can copy this file to the <head> part of your index file
```
  <link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,300;0,400;0,500;1,500&display=swap" rel="stylesheet">

```

You can then use any of these in your css file

font-family: 'Crete Round', serif;
font-family: 'Poppins', sans-serif;

**Comment by sambeck87** [comment]
@Qoosim Don´t worry about it.
Could you help me with the 'roboto' font-family too, please? I need it for the logo and the paragraph.

Thanks for all your help.

**Comment by Qoosim** [comment]
Hi @sambeck87, you can simply check for any one [here](https://fonts.google.com/)

File: style.css [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+section p {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-top: 35px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
```
Comment (MoRaad97) [inline style.css]: - [ ] Kindly try to reduce the spacing like the previous reviewer asked you, I suggest reducing the **margin-top**
ScreenShot_20221027235719.

**Review by MoRaad97 — CHANGES_REQUESTED** [review]
Hi @sambeck87, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
## Hi @sambeck87 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Good job for using the right GitHub Flow 💯
Awesome file organization 🗃️
Commits are descriptive and meaningful 👍
Linters are passing 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @oyepriscilla in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sambeck87** [comment]
@Qoosim @MoRaad97 @OyePriscilla Thank you for your advice and help

**Pull request merged**